projects/plugins/crm/CHANGELOG.md

```diff
@@ -5,11 +5,39 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.5.4-alpha] - 2023-02-15
+### Added
+- adds the necessary migration to move all files that were inside the zbscrm-store folder with a flat struture to the new jpcrm-storage folder that uses a hierarchical structure [#28350]
+- Copy tests from old repo. [#28354]
```

Comment:
From the plugins PoV, then this isn't a new addition but it sounds like it for the end-user 😋